third_party/workspace.bzl

```diff
@@ -138,3 +138,12 @@ def tensorboard_workspace(name = ""):
         ],
         licenses = ["notice"],  # Apache 2.0
     )
+
+    java_import_external(
+        name = "org_jsoup_external",
+        jar_sha256 = "436adf71fe9f326e04fe134cd2785b261f0f4b9b60876adda1de3b6919463394",
```

Comment:
Curious, did you download the jar file and compute this? Or where did it come from?

Reply:
I downloaded the jar and computed it